doc/browsing_cubes/overlay.ipynb

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
It would be nice to have a very brief introduction at the start of the notebook, something along the lines of ...
 "Plotting Overlay Example.  
This example demonstrates how to easily display data from two Iris cubes by overlaying plots of the data using the Cube Browser."